CHANGELOG.md

```diff
@@ -28,3 +28,6 @@
  - Add currency_code field to cart currency data
  - Dispatch sales_quote_save_after event for active (parent) quotes only
  - Fixed consistency for Amasty Gift Card module
+## [v1.1.9](https://github.com/BoltApp/bolt-magento2/releases/tag/1.1.9) 2019-01-24
+ - Allow empty emails in shipping_and_tax API
+ - Add feature to optionally not inject JS on non-checkout pages
```

Comment:
sent store order notifications to email collected from Bolt checkout
create order from parent quote
do not cache empty shipping options array

Reply:
Updated and `git commit --amend`-ed